test/functional/mempool_compatibility.py

```diff
@@ -48,11 +46,10 @@ def run_test(self):
         # unbroadcasted_tx won't pass old_node's `MemPoolAccept::PreChecks`.
         self.connect_nodes(0, 1)
         self.sync_blocks()
-        recipient = old_node.getnewaddress()
         self.stop_node(1)
 
         self.log.info("Add a transaction to mempool on old node and shutdown")
-        old_tx_hash = old_node.sendtoaddress(recipient, 0.0001)
+        old_tx_hash = new_wallet.send_self_transfer(from_node=old_node)["txid"]
```

Comment:
26640: tests fail here because `testmempoolaccept` used by `send_self_transfer` internally was introduced only in [18.0.1](https://github.com/dashpay/dash/blob/master/doc/release-notes/dash/release-notes-18.0.1.md?plain=1#L151). You have to bump `150000, # oldest version supported by the test framework` above to make it work I think.